test/specs/exampleUsingProject.e2e.ts

```diff
@@ -0,0 +1,15 @@
+import { step } from 'mocha-steps';
+import { TestSetup } from '../testSetup.ts';
+import * as utilities from '../utilities/index.ts';
+
+describe('an example using existing project', async () => {
+  let testSetup: TestSetup;
+
+  step('Set up the testing environment', async () => {
+    testSetup = new TestSetup('usingExistingProject');
+    await testSetup.setUp('developer', utilities.repoKeywords.DeployInv);
```

Comment:
You need to set up the test by adding the repo name. After this, you will have the project you want open in VSCode

Reply:
if you do not need to use scratch orgs, you can still set up the test by using setUpTestingEnvironment and createOrUseProject to initialize your test